### PR TITLE
feat(Animea): add new presence

### LINKED ISSUE #6151: Animea | animea.site
### Website name

Animea

### Website URL

animea.site

### Website logo

https://cdn.discordapp.com/attachments/315622468662460416/967856531594965042/Sans_titre.png

### Prerequisites

- [ ] It is a paid service
- [ ] It displays NSFW content
- [ ] It is region restricted

### Description

Animea is a streaming site dedicated to anime and anime movies in HD and free.

### PR DESCRIPTION
**Pull request by RisingSunLight42**
## Description 

Resolves #6151 by adding the requested presence.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

animea_proof_5

animea_proof_6

animea_proof_7

animea_proof_4

animea_proof_3

animea_proof_2

animea_proof_1

</details>

### CODE REVIEW

**Comment by RisingSunLight42** [comment]
@Kozou4ever peux-tu check la langue s'il te plaît ? Vu que c'est que du français, si tu pouvais faire ça pour que l'on gagne un peu de temps et qu'Encrypted ou Bas n'aient pas à demander à quelqu'un de check

File: websites/A/Animea/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "RisingSunLight",
+		"id": "240521747852558347"
+	},
+	"service": "Animea",
+	"description": {
+		"en": "Watch films in HD with language options, dubbing and subtiles.",
+		"fr": "Regardez des films en qualité HD avec des options de langue avec doublage et sous-titres."
```
Comment (Kozou4ever) [inline websites/A/Animea/dist/metadata.json:10]:
```suggestion
		"fr": "Regardez des films en HD avec des options de langues, doublage et sous-titres."
```

File: websites/A/Animea/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "RisingSunLight",
+		"id": "240521747852558347"
+	},
+	"service": "Animea",
+	"description": {
+		"en": "Watch films in HD with language options, dubbing and subtiles.",
```
Comment (Kozou4ever) [inline websites/A/Animea/dist/metadata.json:9]:
```suggestion
		"en": "Watch films in HD with language options, dubbing and subtitles.",
```
There's a typo here

File: websites/A/Animea/presence.ts [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+			presence.getSetting<boolean>("cover"),
+			presence.getSetting<boolean>("buttons"),
+		]);
+
+	switch (pathnameArray[1]) {
+		case "":
+			presenceData.details = "Parcours la page d'accueil";
+			break;
+		case "discovery": {
+			presenceData.details = "Découvre de nouvelles animations";
```
Comment (Kozou4ever) [inline websites/A/Animea/presence.ts:34]:
```suggestion
			presenceData.details = "Découvre de nouveaux animes";
```

**Review by Kozou4ever — CHANGES_REQUESTED** [review]

File: websites/A/Animea/presence.ts (hunk 3 above)
Comment (RisingSunLight42) [reply websites/A/Animea/presence.ts]: Resolved in 401f95334db21f79fa5f8bbe9d999798d7fab508

File: websites/A/Animea/dist/metadata.json (hunk 2 above)
Comment (RisingSunLight42) [reply websites/A/Animea/dist/metadata.json]: Resolved in 401f95334db21f79fa5f8bbe9d999798d7fab508

File: websites/A/Animea/dist/metadata.json (hunk 1 above)
Comment (RisingSunLight42) [reply websites/A/Animea/dist/metadata.json]: Resolved in 401f95334db21f79fa5f8bbe9d999798d7fab508

**Comment by RisingSunLight42** [comment]
Idk why, but Github don't want me to re-request review to Kozou, so @Kozou4ever could you approve my changes ? xD

**Review by Kozou4ever — APPROVED** [review]

**Comment by RisingSunLight42** [comment]
Commit bbac9f1af128ed85ad2cbfeab0650edea680e9f0 remove if statements for settings to put just one at the end of the file to avoid repetition and have a better readability, thanks to @DarkVillager for the suggestion !

**Review by EncryptedDev — APPROVED** [review]

**Review by DarkVillager — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Comment by Tigrouu** [comment]
Hello, I would like to know if it was possible to change the url to animea.ml instead of animea.site

**Comment by DarkVillager** [comment]
> Hello, I would like to know if it was possible to change the url to animea.ml instead of animea.site and possible to add when you see the films

@RisingSunLight42

**Comment by Tigrouu** [comment]
Hello I am redoing the domain change request under this domain animea.ovh @RisingSunLight42

**Pull request merged**